apps/src/templates/sectionAssessments/MultipleChoiceOverviewTable.jsx

```diff
@@ -84,84 +71,68 @@ class MultipleChoiceOverviewTable extends Component {
-            width: 90,
-          }},
-        },
-        cell: {
-          format: answerOptionsOneFormatter,
-          props: {
-            style: {
-            ...tableLayoutStyles.cell,
-            width: 90,
-          }}
-        }
+  getNotAnsweredColumn = (index) => (
+    {
+      header: {
+        label: commonMsg.notAnswered(),
+        props: {style: tableLayoutStyles.headerCell},
       },
-      {
-        property: 'percentAnsweredOptionTwo',
-        header: {
-          label: commonMsg.answerOptionB(),
-          props: {
-            style: {
-            ...tableLayoutStyles.headerCell,
-            width: 90,
-          }},
-        },
-        cell: {
-          format: answerOptionsTwoFormatter,
-          props: {
-            style: {
-            ...tableLayoutStyles.cell,
-            width: 90,
-          }}
-        }
+      cell: {
+        props: {style: tableLayoutStyles.cell},
+      }
+    }
+  );
+
+  getAnswerColumns = (index) => (
+    {
+      header: {
+        label: alphabetMapper[index],
+        props: {style: tableLayoutStyles.headerCell},
       },
-      {
-        property: 'notAnswered',
-        header: {
-          label: commonMsg.notAnswered(),
-          props: {
-            style: {
-            ...tableLayoutStyles.headerCell,
-            width: 90,
-          }},
-        },
-        cell: {
-          format: notAnsweredFormatter,
-          props: {
-            style: {
-            ...tableLayoutStyles.cell,
-            width: 90,
-          }}
-        }
+      cell: {
+        format: answerColumnsFormatter,
+        props: {style: tableLayoutStyles.cell},
+      }
+    }
+  );
+
+  getQuestionColumn = (sortable) => (
+    {
+      header: {
+        label: commonMsg.question(),
+        props: {style: tableLayoutStyles.headerCell},
+        transforms: [sortable],
       },
-    ];
-    return dataColumns;
+      cell: {
+        props: {style: tableLayoutStyles.cell},
+      }
+    }
+  );
+
+  getColumns = (sortable) => {
+    const maxAnswerChoicesLength = this.props.questionAnswerData.reduce((answersTotal, currentAnswerCount) => {
+      if (currentAnswerCount.answers.length > answersTotal) {
+        return currentAnswerCount.answers.length;
+      }
+      return answersTotal;
+    }, 0);
+
+    let dataColumns = [];
+    let columns = this.getQuestionColumn(sortable) ;
+
+    for (let i = 0; i < maxAnswerChoicesLength; i++) {
+      let questionOption = this.getAnswerColumns(i - 1);
+      if (i === 0) {
+        dataColumns.push({property: 'question', ...columns});
+      } else {
+        dataColumns.push({property: 'percentAnswered' , ...questionOption});
+      }
+    }
+      // Add 2 to maxAnswerChoicesLength to ensure notAnswered is the last column.
+      // maxAnswerChoicesLength does not include Question column.
+        dataColumns.push({property: 'notAnswered', ...this.getNotAnsweredColumn(maxAnswerChoicesLength + 2)});
```

Comment:
You've got some extra indentation on these three lines.

Reply:
Fixed!